src/filesystem/node-fs.ts

```diff
@@ -101,7 +101,7 @@ export class NodeFS implements FileSystem {
     return new NodeFS(this.scope, abs)
   }
 
-  async fetch(remote: string, dest: string) {
+  async fetch(remote: string, dest: string, subgroup = false) {
```

Comment:
@Morgul could you use an optional `options` object? adding positional arguments is not really scalable after one or two. in this case, if we add another option in the future, then providing that option would require users to know and provide `subgroup` as well, increasing usage hassle.

Reply:
Sure, that makes complete sense. Just pushed a version with an options object.